specs/rigil/precompiles.md

```diff
@@ -50,178 +57,376 @@ A list of available precompiles in Rigil are as follows:
 
 ### `IsConfidential`
 
-[🔗 Implementation](https://github.com/flashbots/suave-geth/blob/b328d64689930a40eae0a6e834805f3feab6b58f/core/vm/contracts_suave.go#L43)
```

Comment:
why does the PR remove all the implementation links?

Reply:
Well, the PR is generated with [this](https://github.com/flashbots/suave-geth/pull/138) and it is pretty hard to figure out in which line the precompile is implemented with the script.

Reply:
i see! i guess it's not strictly necessary, although nice to have and nice to point people the right direction if they're curious. but makes sense that this is not a great use of time to fix. perhaps this page should simply have a link to the implementations at `contracts_suave.go`?